File: 프로그래머스/lv0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.py
def solution(my_string):
    num_list = []
    numm =['0','1','2','3','4','5','6','7','8','9']
    for i in my_string:
        if i in numm:
            num_list.append(int(i))
    num_list.sort()
    
            
    
    return num_list

File: 프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.js
function solution(my_string) {
    var answer = [];
    let str_li = Array.from(my_string,Number);
    for (let i=0;i<str_li.length;i++){
        if(!isNaN(parseInt(str_li[i] ))){
            answer.push(str_li[i]);
        }
    }
    answer.sort(function(a, b)  {
  return a - b;
});
    return answer;
}